OpenTrans.net/Party.cs

```diff
@@ -125,6 +125,11 @@ public class Party
         /// </summary>
         public string PhoneNo { get; set; }
 
+        /// <summary>
+        /// Additional phone numbers all read / written as the XML "PHONE" element
+        /// </summary>
+        public List<string> AdditionalPhoneNo { get; set; } = new List<string>();
```

Comment:
can you please use a single property for all phone nos?
Furthermore, as you are working this anyway, could you encorporate the three valid phone number types?

![grafik](https://github.com/stephanstapel/OpenTrans.net/assets/2912080/6015c441-3d96-424e-a95f-2e543f61d713)

In this small case, imho it is fine to read all other types as Unknown.